src/librustc_driver/lib.rs

```diff
@@ -198,7 +198,7 @@ pub fn get_trans(sess: &Session) -> Box<TransCrate> {
             rustc_trans_utils::trans_crate::MetadataOnlyTransCrate::new(&sess)
         }
         Some(filename) if filename.starts_with("/") => {
-            rustc_trans_utils::trans_crate::ExternTransCrate::new(&sess, filename)
+            rustc_trans_utils::trans_crate::link_extern_backend(&sess, filename)
```

Comment:
Hmm, can you move that function here? I think `rustc_driver` should be the one with the `libloading` dependency.

Reply:
Oh, if you move that here you can get rid of the `libloading` dependency by using `rustc_metadata::dynamic_lib`.

Reply:
Didnt know about `rustc_metadata::dynamic_lib`.